partials/_general.scss

```diff
@@ -2,106 +2,17 @@ body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100%;
 }
 
-* {
-  scroll-behavior: smooth;
-}
-
-.header {
-  height: 48px;
-  font-weight: 400;
-  font-size: $nav-font-size;
-
-  @include media(">=md") {
-    height: 56px;
-  }
-
-  @include media(">=lg") {
-    height: 125px;
-  }
-}
-
-.hamburger {
-  position: absolute;
-  top: 0;
-  right: 0;
-  margin: 0;
-
-  @include media(">=lg") {
-    display: none;
-  }
-
-  &__img {
-    margin: 16px 16px 0px;
-    cursor: pointer;
-
-    @include media(">=md") {
-      margin: 19px 16px 0px;
-    }
-  }
-}
-
-.nav {
-  line-height: 20px;
-  color: $tws_black;
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-start;
-  gap: 10px;
-  width: 100%;
-  position: relative;
-  height: 48px;
-  font-family: $font-regular;
-
-  @include media(">=md") {
-    height: 56px;
-  }
-
-  @include media(">=lg") {
-    line-height: 24px;
-    height: 61px;
-  }
-
-  &__column {
-    width: 33.33%;
-  }
+body.active {
```

Comment:
this seems like a modifier (&--active)

Reply:
How would you change this to ensure it worked if the active class was now a modifier?